github-pull-requests.go

```diff
@@ -386,34 +79,27 @@ func populatePRs(url string) {
 	htmlData, err := ioutil.ReadAll(resp.Body)
 	exitOnErr(err)
 
+	const layout = "Jan 2, 2006 at 3:04pm (PST)"
```

Comment:
RFC1123     = "Mon, 02 Jan 2006 15:04:05 MST"  looked less friendly than what i have above in the ux. Hence i picked this. I am open to revisiting this later.  I don't really need day info.